app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -3,13 +3,21 @@
 module V0
   class DisabilityCompensationFormsController < ApplicationController
     before_action { authorize :evss, :access? }
+    before_action :validate_name_part, only: [:suggested_conditions]
 
     def rated_disabilities
       response = service.get_rated_disabilities
       render json: response,
              serializer: RatedDisabilitiesSerializer
     end
 
+    def suggested_conditions
+      results = DisabilityContention.suggested(params[:name_part])
+      render json: results,
+             serializer: CollectionSerializer,
```

Comment:
Do you need to specify serializer: CollectionSerializer if you specify each_serializer?